arduino/drive/MotorArduino.ino

```diff
@@ -0,0 +1,39 @@
+#include <Wire.h>
```

Comment:
General comment. Think we have a consistent style guide to our code. This is my suggestion for C++

[Google C++](https://google.github.io/styleguide/cppguide.html)